### PR TITLE
Initial commit for CI

### PR DESCRIPTION
**Pull request by lucernae**
- Integrate Travis
- Integrate codecov

### CODE REVIEW

**Comment by lucernae** [comment]
Weird, travis seems to be building a queue of draft PR. So now, the build is trying to catch up.

Anyway, what needs to be done:

- [ ] Fix flake error. It made travis unhappy
- [x] Secret.py does not automatically generated. It's blocking everything.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - stage: Test
+      script:
+        - pushd deployment
+        - echo "Preparing Docker Stack"
+        - make build up wait-for-db migrate collectstatic
+        - make status
+        - echo "Docker stack ready"
+        - echo "Preparing unittests"
+        - make flake8
+        - echo "Run unittests"
```
Comment (pierrealixt) [inline .travis.yml:42]: `make test` is already present

Comment (lucernae) [reply .travis.yml:42]: Yes, I saw that you create another Make command, so I didn't make a new one here to avoid conflict.

**Comment by lucernae** [comment]
I'm done with this PR @pierrealixt @Christiaanvdm . We should Review + Merge this ASAP to avoid further conflicts with current develop. Improvements should be done in separate PR. This PR contains basic mechanism of running Travis CI config.

It will still be red, since most of the test doesn't pass yet. But we should make separate PR to fix each test.

**Review by pierrealixt — APPROVED** [review]
LVGTM

**Pull request merged**